metadata-ingestion/src/datahub/emitter/mcp.py

```diff
@@ -28,6 +29,31 @@ def _make_generic_aspect(codegen_obj: DictWrapper) -> GenericAspectClass:
     )
 
 
+def _try_from_generic_aspect(
+    aspectName: Optional[str],
+    aspect: Optional[GenericAspectClass],
+) -> Tuple[bool, Optional[_Aspect]]:
+    # The first value in the tuple indicates the success of the conversion,
+    # while the second value is the deserialized aspect.
+
+    if aspect is None:
+        return True, None
+    assert aspectName is not None, "aspectName must be set if aspect is set"
+
+    if aspect.contentType != "application/json":
```

Comment:
we should already have a constant for this somewhere, no?

Reply:
nope but I'll add it